server/apps/immich/src/utils/disk-storage.ts

```diff
@@ -1,12 +1,12 @@
 import { Request } from 'express';
 import { ParamsDictionary } from 'express-serve-static-core';
+import * as mkdirp from 'mkdirp';
```

Comment:
I think we can use native nodeJs API to create directory recursively without adding dependency. https://joshtronic.com/2021/01/17/recursively-create-directories-with-nodejs/

Please be mindful of adding extra dependencies, we only want to add dependency when it is really necessary

Reply:
I just copy cat from multer. I also think `fs.mkdirSync('./path/to/my/directory', { recursive: true })` is suitable. Give me a second.